.travis.yml

```diff
@@ -1,35 +1,9 @@
 sudo: false
-cache: bundler
 language: ruby
 dist: trusty
 rvm:
-  - jruby-9.2.0.0
-  - jruby-head
-  - 2.2.10
-  - 2.3.7
-  - 2.4.4
-  - 2.5.1
-  - ruby-head
-env:
-  global:
-    - JRUBY_OPTS='-J-Xmx1000M'
-    # Travis CI container has two cores, but Ruby can see 32 cores unfortunately.
-    # See https://github.com/rubocop-hq/rubocop/pull/5719
-    #     https://docs.travis-ci.com/user/reference/overview/#Virtualisation-Environment-vs-Operating-System
-    - TEST_QUEUE_WORKERS=2
-  matrix:
-    - 'TASK=spec'
-    - 'TASK=ascii_spec'
-    - 'TASK=internal_investigation'
-matrix:
-  allow_failures:
-    - rvm: ruby-head
-    - rvm: jruby-head
-  fast_finish: true
-before_install:
-  - gem update --system
-  - gem update --remote bundler
+  - 2.4.1
```

Comment:
@bquorning I have a question. Is there a intention to select `2.4.1`? (it is a old Ruby version)

Reply:
> I wanted the Travis no-op build to be as fast as possible

Make sense! I understand that it is based on pre-installation of Travis CI. Thanks for your explanation 🙏